layouts/partials/header.hbs

```diff
@@ -23,11 +23,11 @@
       </ul>
     </nav>
 
-  <svg id="lang-picker-toggler" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
-    <rect id="lang-rect" width="512" height="512" x="0" y="0" fill="none" pointer-events="fill"/>
```

Comment:
I don't think slash of self-close tag can be omitted. 🤔

Reply:
Of course it can :)

Reply:
Actually, you are right. Let me try something and I'll sort this out.

Reply:
Ok fixed. Not sure how I got confused here :/

Reply:
👍